src/server/rest/v1/service/ChargingStationService.ts

```diff
@@ -1022,7 +1030,7 @@ export default class ChargingStationService {
       }
       // Get Transaction
       const transaction = await TransactionStorage.getTransaction(req.user.tenantID, filteredRequest.args.transactionId);
-      UtilsService.assertObjectExists(action, transaction, `Transaction ID '${filteredRequest.args.transactionId}' does not exist`,
+      UtilsService.assertObjectExists(action, transaction, `Transaction ID '${String(filteredRequest.args.transactionId)}' does not exist`,
```

Comment:
toString()

Reply:
In this case filteredRequest.args.transactionId has the type any. I cannot use toString()